pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java

```diff
@@ -100,6 +114,10 @@ public LocalBookkeeperEnsemble(int numberOfBookies, int zkPort, int bkBasePort,
     ServerConfiguration bsConfs[];
     Integer initialPort = 5000;
 
+    // Stream/Table Storage
+    StreamStorageLifecycleComponent streamStorage;
+    Integer streamStoragePort = 4181;
```

Comment:
To make this value configurable seems better.

Reply:
yeah good point, will address that

Reply:
make it configurable from the constructor.

Reply:
Thanks!
Should we add an option like
```
@Parameter(names = { "--stream-storage-port" }, description = "Local bookies stream storage port")
```
to `PulsarStandaloneStarter`?

Reply:
nice point. I added a starter parameter.

Reply:
Thanks! 👍